src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -478,18 +478,27 @@ function GlobeControls(camera, domElement, engine) {
         }
     };
 
-    var getPickingPosition = (function getGetPickingPositionFn() {
-        var engineGfx = engine;
-        var position;
+    const getPickingPosition = (function getGetPickingPositionFn(pickingHelper) {
+        const engineGfx = engine;
+        let position;
+
+        return function getPickingPosition(coords) {
+            if (enableTargetHelper) {
+                pickingHelper.visible = false;
+                cameraTargetOnGlobe.visible = false;
+            }
 
-        return function getPickingPosition(coords)
-        {
             position = engineGfx.getPickingPositionFromDepth(coords);
             engineGfx.renderScene();
 
+            if (enableTargetHelper) {
+                pickingHelper.visible = true;
+                cameraTargetOnGlobe.visible = true;
+            }
```

Comment:
if there are  targets helper, we hide targets helper before picking
and after picking if there are  targets helper we display targets helper for rendering